test/test_cases/test_case_setup.sh

```diff
@@ -1,14 +1,31 @@
 #!/bin/bash
 set -euxo pipefail
 
-# TODO: replace the following with `$cli create secret`
-$cli secrets new-dockercfg dockerpullsecret \
-      --docker-server=${DOCKER_REGISTRY_PATH} \
+announce "Creating image pull secret."
+
+if [[ "${PLATFORM}" == "kubernetes" ]]; then
+     $cli delete --ignore-not-found secret dockerpullsecret
+
+     $cli create secret docker-registry dockerpullsecret \
+      --docker-server=$DOCKER_REGISTRY_URL \
       --docker-username=_ \
-      --docker-password=$($cli whoami -t) \
+      --docker-password=_ \
       --docker-email=_
 
-$cli secrets add serviceaccount/default secrets/dockerpullsecret --for=pull
+elif [[ "$PLATFORM" == "openshift" ]]; then
+
+    $cli delete --ignore-not-found secrets dockerpullsecret
+
+    # TODO: replace the following with `$cli create secret`
```

Comment:
can we do this TODO easily?

Reply:
I did not write this one- just reformated. @moticless you have input here?